packages/comms/src/services/wsStore.ts

```diff
@@ -0,0 +1,230 @@
+        Key: string;
+        Value: string;
+    }
+
+    export interface Pairs2 {
+        Pair: Pair2[];
+    }
+
+    export interface FetchKeyMDResponse {
+        Exceptions: Exceptions;
+        StoreName: string;
+        Namespace: string;
+        Key: string;
+        Pairs: Pairs2;
+    }
+
+    export interface KeySet {
+        Key: string[];
+    }
+
+    export interface ListKeysResponse {
+        Exceptions: Exceptions;
+        StoreName: string;
+        Namespace: string;
+        KeySet: KeySet;
+    }
+
+    export interface Namespaces {
+        Namespace: string[];
+    }
+
+    export interface ListNamespacesResponse {
+        Exceptions: Exceptions;
+        StoreName: string;
+        Namespaces: Namespaces;
+    }
+
+    export interface WsstorePingResponse {
+    }
+
+    export interface SetResponse {
+        Exceptions: Exceptions;
+        Success: boolean;
+    }
+}
+
+export class StoreService extends Service {
+
+    constructor(optsConnection: IOptions | IConnection) {
+        super(optsConnection, "WsStore", "1");
+    }
+
+    CreateStore(request: WsStore.CreateStoreRequest): Promise<WsStore.CreateStoreResponse> {
+        return this._connection.send("Fetch", request);
+    }
+
+    Delete(request: WsStore.DeleteRequest): Promise<WsStore.DeleteResponse> {
+        return this._connection.send("Delete", request).catch(e => {
+            if (e.isESPExceptions && e.Exception.some(e => e.Code === -1)) {
+                //  "Delete" item does not exist  ---
+                return {
+                    Exceptions: undefined,
+                    Success: true
+                } as WsStore.DeleteResponse;
+            }
+            throw e;
+        });
+    }
+
+    DeleteNamespace(request: WsStore.DeleteNamespaceRequest): Promise<WsStore.DeleteNamespaceResponse> {
+        return this._connection.send("DeleteNamespace", request);
+    }
+
+    Fetch(request: WsStore.FetchRequest): Promise<WsStore.FetchResponse> {
+        return this._connection.send("Fetch", request).catch(e => {
+            if (e.isESPExceptions && e.Exception.some(e => e.Code === -1)) {
+                //  "Fetch" item does not exist  ---
+                return {
+                    Exceptions: undefined,
+                    Value: undefined
+                } as WsStore.FetchResponse;
+            }
+            throw e;
+        });
+    }
+
+    FetchAll(request: WsStore.FetchAllRequest): Promise<WsStore.FetchAllResponse> {
+        return this._connection.send("FetchAll", request);
+    }
+
+    FetchKeyMD(request: WsStore.FetchKeyMDRequest): Promise<WsStore.FetchKeyMDResponse> {
+        return this._connection.send("FetchKeyMD", request);
+    }
+
+    ListKeys(request: WsStore.ListKeysRequest): Promise<WsStore.ListKeysResponse> {
+        return this._connection.send("ListKeys", request);
+    }
+
+    ListNamespaces(request: WsStore.ListNamespacesRequest): Promise<WsStore.ListNamespacesResponse> {
```

Comment:
Are these methods wrappers for existing services?

Reply:
Yes all the files in the "services" folder are generated from existing services (well the request/response structures are).